strongbox-web-core/src/test/java/org/carlspring/strongbox/rest/ArtifactControllerTest.java

```diff
@@ -0,0 +1,761 @@
+ */
+
+@RunWith(SpringJUnit4ClassRunner.class)
+@ContextConfiguration(classes = WebConfig.class)
+@WebAppConfiguration
+public class ArtifactControllerTest
+        extends BackendBaseTest
+{
+
+    private static final String TEST_RESOURCES = "target/test-resources";
+
+    private static final File GENERATOR_BASEDIR = new File(ConfigurationResourceResolver.getVaultDirectory() +
+                                                           "/local");
+
+    private static final File REPOSITORY_BASEDIR_RELEASES = new File(ConfigurationResourceResolver.getVaultDirectory() +
+                                                                     "/storages/storage0/releases");
+
+    private static final Logger logger = LoggerFactory.getLogger(ArtifactControllerTest.class);
+
+
+    @Before
+    public void setUpClass()
+            throws Exception {
+        generateArtifact(REPOSITORY_BASEDIR_RELEASES.getAbsolutePath(),
+                         "org.carlspring.strongbox.resolve.only:foo",
+                         "1.1" // Used by testResolveViaProxy()
+        );
+
+        // Generate releases
+        // Used by testPartialFetch():
+        generateArtifact(REPOSITORY_BASEDIR_RELEASES.getAbsolutePath(),
+                         "org.carlspring.strongbox.partial:partial-foo",
+                         "3.1", // Used by testPartialFetch()
+                         "3.2"  // Used by testPartialFetch()
+        );
+
+        // Used by testCopy*():
+        generateArtifact(REPOSITORY_BASEDIR_RELEASES.getAbsolutePath(),
+                         "org.carlspring.strongbox.copy:copy-foo",
+                         "1.1", // Used by testCopyArtifactFile()
+                         "1.2"  // Used by testCopyArtifactDirectory()
+        );
+
+        // Used by testDelete():
+        generateArtifact(REPOSITORY_BASEDIR_RELEASES.getAbsolutePath(),
+                         "com.artifacts.to.delete.releases:delete-foo",
+                         "1.2.1", // Used by testDeleteArtifactFile
+                         "1.2.2"  // Used by testDeleteArtifactDirectory
+        );
+
+        generateArtifact(REPOSITORY_BASEDIR_RELEASES.getAbsolutePath(),
+                         "org.carlspring.strongbox.partial:partial-foo",
+                         "3.1", // Used by testPartialFetch()
+                         "3.2"  // Used by testPartialFetch()
+        );
+
+        generateArtifact(REPOSITORY_BASEDIR_RELEASES.getAbsolutePath(),
+                         "org.carlspring.strongbox.browse:foo-bar",
+                         "1.0", // Used by testDirectoryListing()
+                         "2.4"  // Used by testDirectoryListing()
+        );
+
+        //noinspection ResultOfMethodCallIgnored
+        new File(TEST_RESOURCES).mkdirs();
+    }
+
+
+    @Test
+    @WithUserDetails("admin")
+    public void testUserAuth()
+            throws Exception
+    {
+
+        String url = getContextBaseUrl() + "/storages/greet";
+
+
+        given()
```

Comment:
Could you please align such multi-line blocks by the `.` like for example:

        given().contentType(ContentType.JSON)
               .when()
               .get(url)
               .then()
               .statusCode(200)
               .body(containsString("success"))
               .toString();

I've seen this across a number of places, so it would be great, if you could fix it there as well.